app/models/invoice.rb

```diff
@@ -4,4 +4,8 @@ class Invoice < ApplicationRecord
   has_many :invoice_items
   has_many :transactions
   has_many :items, through: :invoice_items
+
+  def self.random
+    order("RANDOM()").first
+  end
```

Comment:
really nice job of finding this query. It's tempting to use Ruby but I'm sure this is much more efficient.